docs/using-kontena/loadbalancer.md

```diff
@@ -103,7 +103,7 @@ See HAProxy [docs](http://cbonte.github.io/haproxy-dconv/1.6/configuration.html#
 
 ## Stats
 
-Kontena Load Balancer exposes a statistics web UI only on the private overlay network. To access the statistics you must use the [VPN](vpn-access.md) to access the overlay network. The statistics are exposed on port 1000 of the Kontena Load Balancer instances.
+Kontena Load Balancer exposes a statistics web UI only on the private overlay network. To access the statistics you must use the [VPN](vpn-access.md) to access the overlay network. The statistics are exposed on port 1000 of the Kontena Load Balancer instances. You can also expose the port 1000 using the ports option in `kontena.yml`.
```

Comment:
Maybe add a note that if exposing the stats to public networks one should really override the default passwd with strong password.